grakn-kb/src/main/java/ai/grakn/kb/internal/GraknTxAbstract.java

```diff
@@ -126,6 +129,18 @@
         }
     }
 
+    private static final Method queryRunnerFactory;
+
+    static {
```

Comment:
can we add an explanation about the intention of this `static` block? is `method == null` an expected, normal condition?

Reply:
does it make sense to just create a private method for this operation:

```
private static final Method queryRunnerFactory = getCreateMethodFromClass(QUERY_RUNNER_CLASS_NAME);

private Method getCreateMethodFromClass(String className) {
        Method method;
        try {
            method = Class.forName(QUERY_RUNNER_CLASS_NAME).getDeclaredMethod("create", GraknTx.class);
        } catch (NoSuchMethodException | SecurityException | ClassNotFoundException e) {
            method = null;
        }
        return method;
}
```

Reply:
The private method is a good idea!